.github/workflows/deploy-development-docker-image.yml

```
@@ -0,0 +1,44 @@
+name: Create and publish development Docker image
+
+on:
+  push:
+    branches-ignore:
+      - "main"
```

if its the development docker image, why ignore main, and not just on development branch? now it creates a new image on every branch push no?
```suggestion
      - "development"
```